### PR TITLE
Add coverage badge.

### PR DESCRIPTION
**Pull request by godofredoc**
We are finally generating coverage reports on post submit for the flutter repository. This is adding the badge to the README file of the repository.

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X] I signed the [CLA].
- [X] I listed at least one issue that this PR fixes in the description above.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I added new tests to check the change I am making, or this PR is [test-exempt].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 [![Build Status - Cirrus][]][Build status]
 [![Discord badge][]][Discord instructions]
 [![Twitter handle][]][Twitter badge]
+[![codecov](https://codecov.io/gh/flutter/flutter/branch/master/graph/badge.svg?token=11yDrJU2M2)](https://codecov.io/gh/flutter/flutter)
```
Comment (christopherfujino) [inline README.md:13]: So it looks like this is only coverage for Flutter tools. Do we still want to show this badge? And if so, we should probably clarify.

Comment (christopherfujino) [reply README.md:13]: https://app.codecov.io/gh/flutter/flutter/tree/master/packages

Comment (christopherfujino) [reply README.md:13]: Actually, we shouldn't land this, we haven't uploaded these reports in years

**Review by christopherfujino — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (christopherfujino) [reply README.md:13]: https://app.codecov.io/gh/flutter/flutter/commits

Comment (godofredoc) [reply README.md:13]: Just fixed that yesterday: https://github.com/flutter/flutter/actions/workflows/coverage.yml we will be generating per commit reports starting in a few mins from now.

Comment (christopherfujino) [reply README.md:13]: Awesome, I see this updated now, LGTM!

**Review by christopherfujino — APPROVED** [review]
Screenshot from 2022-10-07 15-14-05

LGTM

**Pull request merged**